### DISCUSSION TITLE
Failed to load SWC binary for android/arm64

### DISCUSSION THREAD

**Discussion by EAGLE1309** [vercel/next.js#58015] [Help]
### Summary

Just installed NextJs on android using termux.

After doing `npm run dev`

I'm getting the follow error attached in image
![Screenshot_2023-11-04-12-00-57-235-edit_com termux](https://github.com/vercel/next.js/assets/84711967/83863a32-e55b-4fdc-890a-7f8c13e2fb3b)

What should I do to fix it?

Node version: `v20.2.0`

### Additional information

![Screenshot_2023-11-04-12-02-43-701-edit_com termux](https://github.com/vercel/next.js/assets/84711967/e2c8d0d1-5964-421c-99da-d5ddba6a8b8b)

**Comment by mahmood601** [comment]
Is there any progress in this issue 🤔

**Comment by ogtayhuseynov0** [comment]
.
